libs/blocks/tabs/tabs.js

```diff
@@ -101,11 +101,19 @@ function getUniqueId(el, rootElem) {
   return [...tabs].indexOf(el) + 1;
 }
 
+function stopEventBubbling(event) {
+  event.stopPropagation();
+}
+
 function configTabs(config, rootElem) {
   if (config['active-tab']) {
     const id = `#tab-${CSS.escape(config['tab-id'])}-${CSS.escape(getStringKeyName(config['active-tab']))}`;
     const sel = rootElem.querySelector(id);
-    if (sel) sel.click();
+    if (sel) {
+      sel.addEventListener('click', stopEventBubbling, false);
+      sel.click();
+      sel.removeEventListener('click', stopEventBubbling, false);
+    }
```

Comment:
nit - In addition to @Ruchika4 change, inline one liner
```suggestion
    if (sel) {
      sel.addEventListener('click', (e) => e.stopPropagation(), { once: true });
      sel.click();
    }
```

Reply:
Thanks! Done.